testframework/integration/modules/geometry/trishape/test_trishape.py

```diff
@@ -49,7 +49,7 @@
 class TestTriShape(SingleNif):
     """Test base geometry, single blender object."""
 
-    g_path = 'geometry/trishape'  # (documented in base class)
+    g_path = 'geometry/n_geom'  # (documented in base class)
```

Comment:
Is this right?

Reply:
Nope. No clue how that happened. Probably pycharm during trishape refactor.

Reply:
Everything else looks good beside, jut to kick it out

Reply:
done!